Sources/Dependencies/Documentation.docc/Articles/Testing.md

```diff
@@ -168,4 +168,39 @@ transitively get access to it through the app itself. In Xcode, go to "Build Pha
 "Dependencies" from the "Link Binary With Libraries" section. When using SwiftPM, remove the
 "Dependencies" entry from the `testTarget`'s' `dependencies` array in `Package.swift`.
 
+### Test case leakage
+
+Sometimes it is possible to have tests that pass successfully when run in isolation, but somehow
```

Comment:
```suggestion
Sometimes it is possible to have tests that pass successfully when run in isolation, but somehow fail
```
Just noticed the missing word here. Anyways, thanks for adding docs on this! Much appreciated as these will be handy to reference to my team when this situation comes up.

Reply:
good catch, thanks!